### PR TITLE
Adds new Ve'katak Phalanx uniforms + Vaurca flagpatches

### PR DESCRIPTION
**Pull request by feartheblackout**
Adds 2 new uniforms for the Ve'katak Phalanx; the Representative and Reserve uniforms, for Liaisons and Assistants/Off-Duties respectively
Credit to @RustingWithYou for the original sprites, new variants by @GMR25 

Fixes minor spriting issues with the Ve'katak Security and First Responder uniforms (Missing pixels, floating brown pixels)
Credit to @RustingWithYou for the original sprites, fixes by @GMR25 

Adds four new flagpatches (Sedantis, Zo'ra, K'lax, C'thur)
Credit to @GMR25 for the sprites

Fixes minor issues with the Phalanx Faceplate
Removes unintentional SCC access to the proprietary Ve'katak augment, at the request of bug lore
Adds the (missing) ability for Unathi to take the augment

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

File: code/modules/client/preference_setup/loadout/items/accessories.dm [hunk 1]
```diff
@@ -626,6 +626,10 @@
 	flagpatch_national["flagpatch, nralakk"] = /obj/item/clothing/accessory/flagpatch/nralakk
 	flagpatch_national["flagpatch, hegemony"] = /obj/item/clothing/accessory/flagpatch/hegemony
 	flagpatch_national["flagpatch, port antillia"] = /obj/item/clothing/accessory/flagpatch/portantillia
+	flagpatch_national["flagpatch, sedantis"] = /obj/item/clothing/accessory/flagpatch/sedantis
+	flagpatch_national["flagpatch, zo'ra"] = /obj/item/clothing/accessory/flagpatch/zora
+	flagpatch_national["flagpatch, k'lax"] = /obj/item/clothing/accessory/flagpatch/klax
+	flagpatch_national["flagpatch, c'thur"] = /obj/item/clothing/accessory/flagpatch/cthur
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/items/accessories.dm:632]: i know these aren't the first species based flagpatches, but wouldn't this be better locked behind a species selection?

**Review by Arrow768 — APPROVED** [review]

File: code/modules/clothing/factions/pmcg.dm [hunk 2]
```diff
@@ -86,3 +86,18 @@
...
+	desc = "A uniform used by the forces of the Ve'katak Phalanx, a Vaurca-run private military company. This one has ice blue shoulder stripes, identifying it as belonging to a Phalanx member that is representing the direct interests of the Phalanx and their immediate employers."
+	desc_extended = "These uniforms are designed to fit under the combat hardsuits favored by the Phalanx. They are utilitarian in design, and reportedly somewhat uncomfortable - though few of the non-Vaurcae bold enough to join Ve'katak seem to complain."
+	icon = 'icons/clothing/under/uniforms/pmcg.dmi'
+	icon_state = "phalanx-rep_jumpsuit"
+	item_state = "phalanx-rep_jumpsuit"
+	contained_sprite = TRUE
+
+/obj/item/clothing/under/rank/pmc/vekatak_phalanx/reserve
+	name = "\improper Ve'katak Phalanx reserve uniform"
+	desc = "A uniform used by the forces of the Ve'katak Phalanx, a Vaurca-run private military company. This one has green shoulder stripes, identifying it as belonging to a Phalanx member that is not presently serving an active combat role."
```
Comment (GMR25) [inline code/modules/clothing/factions/pmcg.dm:101]:
```suggestion
	desc = "A uniform used by the forces of the Ve'katak Phalanx, a Vaurca-run private military company. This one has green shoulder stripes, identifying it as belonging to a Phalanx member that is not presently serving in an active combat role."
```

**Comment by GMR25** [comment]
The infernal item icons almost escaped my notice for being fixed.

**Pull request merged**